Authentication/LoginProviders/LoginProvider.swift

```diff
@@ -136,4 +140,14 @@ public protocol LoginProvider {
     */
     func logOut() -> SignalProducer<(), LoginProviderErrorType>
     
+    /**
+        Handles the url returned by the login process. Parameters should be the same of the AppDelegate method.
```

Comment:
I would say something like:
```
Handles url opening process:
    if the provider recognizes the url and knows how to react to it, it will return true and proceed to open the url. For the provider to be able to open it correctly, it should receive the same arguments your app's AppDelegate does for this.
    if the provider doesn't recognize the url, it will return false.
```